src/psyclone/tests/config_test.py

```diff
@@ -640,3 +641,54 @@ def test_access_mapping_order(tmpdir):
         api_config = Config.get().api_conf("dynamo0.3")
         for access_mode in api_config.get_access_mapping().values():
             assert isinstance(access_mode, AccessType)
+
+
+def test_psy_data_prefix(tmpdir):
+    ''' Check the handling of PSyData class prefixes.'''
+    config_file = tmpdir.join("config.correct")
+    with config_file.open(mode="w") as new_cfg:
+        new_cfg.write(_CONFIG_CONTENT)
+        new_cfg.close()
+        config = Config()
+        config.load(config_file=str(config_file))
+
+        assert "profile" in config.valid_psy_data_prefix
```

Comment:
`valid_psy_data_prefixes` to indicate it's a list?

Reply:
Of course, stupid me - I called the config file option ... PREFIXES, but not the attribute :(